app/models/profile.rb

```diff
@@ -17,7 +17,6 @@ class Profile < ApplicationRecord # rubocop:disable Metrics/ClassLength
   validates :name, presence: true
   validates :benchmark_id, presence: true
   validates :compliance_threshold, numericality: true
-  validates :account, absence: true, unless: -> { hosts.any? }
```

Comment:
Had to remove this validation because technically you should be able to create policies without any hosts according to the UX flow - see https://github.com/RedHatInsights/compliance-frontend/pull/197 for context